godot-ffi/src/binding/mod.rs

```diff
@@ -79,6 +79,29 @@ unsafe impl Sync for ClassLibraryPtr {}
 // SAFETY: See `Sync` impl safety doc.
 unsafe impl Send for ClassLibraryPtr {}
 
+// ----------------------------------------------------------------------------------------------------------------------------------------------
+
+/// # Safety
+/// The table must not have been initialized yet.
+unsafe fn initialize_table<T>(table: &ManualInitCell<T>, value: T, what: &str) {
+    debug_assert!(
+        !table.is_initialized(),
+        "method table for {what} should only be initialized once"
+    );
+
+    // SAFETY: The table hasn't been initialized yet.
+    unsafe { table.set(value) }
+}
+
+/// # Safety
+/// The table must have been initialized.
+unsafe fn get_table<T>(table: &'static ManualInitCell<T>, msg: &str) -> &'static T {
+    debug_assert!(table.is_initialized(), "{msg}");
+
+    // SAFETY: The table has been initialized.
+    unsafe { table.get_unchecked() }
+}
```

Comment:
Also here, again I don't see how the `// SAFETY` statement + `unsafe` block adds anything.

The same information is now appearing 3 times:
```rs
// # Safety
/// The table must have been initialized.
```
```rs
debug_assert!(table.is_initialized(), "{msg}");
```
```rs
// SAFETY: The table has been initialized.
unsafe { ... }
```

We should avoid bureaucratically following rules. Not only does it take much longer to read the code, but such duplication also has the risk of getting out of sync if something changes, but not all places are updated.

Regarding `unsafe` block, the function has a single logic statement (after the assert), so I also don't see any benefit. Further, it would reintroduce a pointless `// SAFETY` clause if we ever enforce that lint.